src/utils/migrations/0017_auto_20200721_1421.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-07-21 12:07
+from __future__ import unicode_literals
+import re
+
+from django.db import migrations, transaction
+
+def parse_email_addresses(entry):
+    match = re.findall(r'[\w\.-]+@[\w\.-]+\.\w+', entry.to)
+    return match
+
+
+def move_to_m2m(apps, schema_editor):
+    LogEntry = apps.get_model('utils', 'LogEntry')
+    ToAddress = apps.get_model('utils', 'ToAddress')
+
+    email_log_entries = LogEntry.objects.filter(to__isnull=False)
+
+    for entry in email_log_entries:
+        with transaction.atomic():
```

Comment:
Docs suggest that the `Migration` class has to be flagged as not atomic in order to execute parts of the migration within a transaction of their own.
https://docs.djangoproject.com/en/1.11/howto/writing-migrations/#non-atomic-migrations

That said, I don't understand why we run a transaction for each entry, is the length of the table problematic? If so, should we do the same in `move_to_strings`?